recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "sktime" %}
-{% set version = "0.23.0" %}
+{% set version = "0.23.1" %}
 
 package:
   name: sktime-suite
   version: {{ version }}
 
 source:
   url: https://github.com/sktime/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
GitHub archive urls are not stable wrt the hash which is why it's recommended to build from the `pypi` sdist.

Reply:
this was autogenerated - do we need to change this?

Reply:
It's fine -it shouldn't cause any issues. Just mentioning for reference. You can of course switch to `pypi` sources if you want.

* https://github.com/conda-forge/staged-recipes/blob/8f3ea8711660e3df55d1faa15a75d2bc9c0678c1/recipes/example/meta.yaml#L16-L21
* https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/

Reply:
ok, thx for clarifying.